vault-dev-server/src/queue/service.ts

```diff
@@ -35,9 +42,56 @@ class QueueServiceByProject implements QueueServiceImplementation {
     context: CallContext
   ): Promise<{ instanceId?: string | undefined }> {
     const instanceId = v4();
+    // console.log(`Entering initInstance`)
     return { instanceId };
   }
 
+  // private logFilePath = path.join(__dirname, 'workerLogs.json');
+  private logFilePath = 'workerLogs.json';
+
+  async appendLog(entries: any){
+    const data = JSON.stringify(entries, null, 2);
+    const delimiter = this.logEntriesExist ? ',\n' : '';
+    this.logEntriesExist = true;
+    fs.appendFile(this.logFilePath, data);
+  }
+
+  async logWorkers(queue: Queue, projectUuid: string){
+    const workers = await queue.getWorkers();
+    const capacity = workers.length;
+    if(capacity > 0){
+      workers.forEach(worker => {
+        // console.log(`Queue name: ${queue.name}, Worker ID: ${worker.id}, Status: ${worker.status}`);
+        const logEntries = workers.map(worker => ({
+          projectUuid: projectUuid,
+          queueName: queue.name,
+          workerId: worker.id
+        }))
```

Comment:
Missing semicolon. In fact please use the prettier formatter in vscode before commiting.

Reply:
fix in new commit, not make new pr yet